jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java

```diff
@@ -174,6 +180,8 @@ public final long getFreePhysicalMemorySize() {
 	/**
 	 * {@inheritDoc}
 	 */
+	@Deprecated(since = "14")
```

Comment:
getCpuLoad() shouldn't be deprecated.

Reply:
The method is deprecated in the interface `com.ibm.lang.management.OperatingSystemMXBean`.

Reply:
I don't see that getCpuLoad() exists in com.ibm.lang.management.OperatingSystemMXBean.
Looking at https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html it's not deprecated.

Reply:
Corrected and updated to use `getSystemCpuLoadImpl()`.